packages/async-rewriter2/src/async-writer-babel.spec.ts

```diff
@@ -61,7 +61,17 @@ describe('AsyncWriter', () => {
       return runUntranspiledCode(transpiled, context);
     };
     runUntranspiledCode = (code: string, context?: any) => {
-      return vm.runInContext(code, context ?? ctx);
+      return vm.runInContext(code, context ?? ctx, {
+        async importModuleDynamically(specifier: string) {
+          if (specifier === 'forty-two') {
```

Comment:
Omg, there is [actually a package](https://www.npmjs.com/package/forty-two) with that name 😆